pkg/kor/all.go

```diff
@@ -144,9 +148,25 @@ func GetUnusedAll(includeExcludeLists IncludeExcludeLists, clientset kubernetes.
 		allDiffs = append(allDiffs, namespaceIngressDiff)
 		namespacePdbDiff := getUnusedPdbs(clientset, namespace)
 		allDiffs = append(allDiffs, namespacePdbDiff)
+
 		output := FormatOutputAll(namespace, allDiffs)
-		fmt.Println(output)
-		fmt.Println()
+
+		outputBuffer.WriteString(output)
+		outputBuffer.WriteString("\n")
+	}
+
+	if len(slackParams) == 1 {
```

Comment:
instead of relying on the length and order of the passed params I think it would be better to pass a single `slackOpts` struct and check what's set or not there, this also avoids the `if`/`else` chain in the cobra `Run` functions when calling these methods.

Reply:
it should also be possible to consolidate `SendToSlackWebhook` and `SendFileToSlack` into a single `SendToSlack` function and handle everything from there based on what's being set in the `slackOpts` structure.

Reply:
Agreed. Gonna work on these.